### DISCUSSION TITLE
Request for GitHub Actions IP Addresses

### DISCUSSION THREAD

**Discussion by Darshanrajput98** [community/community#55917] [Actions] [answered]
### Select Topic Area

Show & Tell

### Body

Dear GitHub Support Team,

I hope this email finds you well. I am writing to request information regarding the IP addresses used by GitHub Actions.

I am currently working on a project that involves integrating GitHub Actions into our workflow, and for security and compliance purposes, we need to whitelist the IP addresses associated with GitHub Actions in our network infrastructure. This step is necessary to ensure that our systems only allow access from trusted sources.

I kindly request your assistance in providing me with the range or list of IP addresses that GitHub Actions may use. Having this information will allow us to configure our network settings appropriately and ensure a seamless integration of GitHub Actions into our infrastructure.

Your support in this matter would be greatly appreciated. If you require any additional details or have any further questions, please do not hesitate to reach out to me.

Thank you in advance for your attention to this request. I look forward to hearing from you soon.

Best regards,
Darshan Rajput

**Comment by airtower-luna** [comment, marked as answer]
This is the community forum, if you want to reach support you need to use the support form. :wink: 

Anyway, your question is answered here: https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/about-githubs-ip-addresses You can call the [`meta` API endpoint](https://api.github.com/meta) and find the IP ranges for various things (including `actions`) in the response.

**Reply by Darshanrajput98** [reply]
ok , thanks.

**Reply by airtower-luna** [reply]
Glad I could help! Please don't forget to mark the answer. :slightly_smiling_face:
